### ISSUE TITLE
VoiceOver issue: Results not "voiced" when searching for hashtags

### ISSUE THREAD

**Issue by sofquipeut** [feditext/feditext#222]
In the "Explore" tab, when searching for a hashtag, VoiceOver doesn't speak the results found when flicking within them. No problem when searching for something else.

**Comment by VyrCossont** [comment]
Cannot get the Explore tab to list *anything* in the simulator, or even let me select the search results table: it comes up as "Search results: group" but empty and I can't interact with anything. Will try on actual device as well.

Note that we use a [`UISearchController`](https://developer.apple.com/documentation/uikit/uisearchcontroller) for the standard iOS search experience, but it's entirely possible that we're not using it *right*. Worth checking against recent Apple [sample code](https://developer.apple.com/documentation/uikit/view_controllers/displaying_searchable_content_by_using_a_search_controller).
